src/System.Security.Cryptography.Encoding/ref/System.Security.Cryptography.Encoding.cs

```diff
@@ -20,6 +20,49 @@ public partial class AsnEncodedData
         public virtual void CopyFrom(System.Security.Cryptography.AsnEncodedData asnEncodedData) { }
         public virtual string Format(bool multiLine) { return default(string); }
     }
+    public sealed partial class AsnEncodedDataCollection : System.Collections.ICollection, System.Collections.IEnumerable
+    {
+        public AsnEncodedDataCollection() { }
+        public AsnEncodedDataCollection(System.Security.Cryptography.AsnEncodedData asnEncodedData) { }
+        public int Count { get { return default(int); } }
+        public System.Security.Cryptography.AsnEncodedData this[int index] { get { return default(System.Security.Cryptography.AsnEncodedData); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
```

Comment:
Do you know if the genapi tool has been updated? I got mine a while ago from https://www.nuget.org/api/v2/package/Microsoft.DotNet.BuildTools.GenAPI/1.0.0-beta-00081 and it hasn't changed. Thanks